jobs/config.json

```diff
@@ -3150,6 +3150,7 @@
     "args": [
       "--cluster=gce-scale-cluster",
       "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-scalability-highspeed-common.env",
```

Comment:
Where's the file?

Reply:
Good question - forgot to git add it. Should be there now.